runtime/databricks/automl_runtime/forecast/deepar/model.py

```diff
@@ -107,7 +107,10 @@ def predict(self,
         else:
             pred_df = pred_df.drop(columns='item_id')
 
-        pred_df[self._time_col] = pred_df[self._time_col].dt.to_timestamp()
+        if self._frequency_unit == 'W':
```

Comment:
Can you move this logic into a function and add comments? E.g.
```
pred_df = self.normalize_time_column(pred_df)
```
And add unit test for this function.